### PR TITLE
Discipline Priest Checklist

### PR DESCRIPTION
**Pull request by rp4rk**
* Minor change to the rule styling to enforce a visual hierarchy. ( [New](https://user-images.githubusercontent.com/8212881/42739294-ae5e77b8-888c-11e8-96de-0b39eb7583a9.png) | [Old](https://user-images.githubusercontent.com/8212881/42739309-f3993458-888c-11e8-9586-5b78310cd3c1.png) )

* Basics for the discipline checklist

### CODE REVIEW

**Review by MartijnHols — CHANGES_REQUESTED** [review]
The checklist looks really good! The only blocker is the layout changes.

File: src/Parser/Priest/Discipline/Modules/Features/Checklist/Component.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            <AbilityRequirement spell={SPELLS.SHADOW_COVENANT_TALENT.id} />
+          )}
+          <AbilityRequirement spell={SPELLS.PENANCE.id} />
+        </Rule>
+
+        <Rule
+          name="Use cooldowns effectively"
+          description={(
+            <React.Fragment>
+              Cooldowns are an important part of healing, try to use them to counter fight mechanics. For example if a boss has burst damage each 1.5 minutes, <SpellLink id={SPELLS.RAPTURE.id} /> should be used to counter it.
```
[the remark this answers is not in the document]
Comment (rp4rk) [reply src/Parser/Priest/Discipline/Modules/Features/Checklist/Component.js]: I'd tend to say each but every reads better here :D

File: src/Parser/Priest/Discipline/Modules/Features/Checklist/Component.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      <Checklist>
+        <Rule
+          name="Use core abilities as often as possible"
+          description={(
+            <React.Fragment>
+              Using your core abilities as often as possible will typically result in better performance, remember to <SpellLink id={SPELLS.SMITE.id} /> as often as you can!
+            </React.Fragment>
+          )}
+        >
+          {combatant.hasTalent(SPELLS.SCHISM_TALENT.id) && (
```
[the remark this answers is not in the document]
Comment (rp4rk) [reply src/Parser/Priest/Discipline/Modules/Features/Checklist/Component.js]: Yeah already tried that, better off with this for now I think.

File: src/Parser/Core/Modules/Features/Checklist2/Rule.js [hunk 3]
```diff
@@ -117,22 +117,20 @@ class Rule extends React.PureComponent {
...
+          <div className="details" style={{ display: this.state.expanded ? undefined : 'none', paddingLeft: 65, position: 'relative', minHeight: 80 }}>
             {description && (
-              <div className="row" style={{ marginBottom: 10 }}>
-                <div className="col-md-12 text-muted">
-                  <div className="flex">
-                    <div className="flex-sub content-middle" style={{ fontSize: '3.5em', lineHeight: 1, marginRight: 20 }}>
-                      <div>{/* this div ensures vertical alignment */}
-                        <InformationIcon />
+              <React.Fragment>
+                <InformationIcon style={{ position: 'absolute', top: 15, left: 18, fontSize: '2em' }} />
```
[the remark this answers is not in the document]
Comment (rp4rk) [reply src/Parser/Core/Modules/Features/Checklist2/Rule.js]: I'll get it lined up

File: src/Parser/Core/Modules/Features/Checklist2/Rule.js [hunk 4]
```diff
@@ -117,22 +117,20 @@ class Rule extends React.PureComponent {
             </div>
           </div>
           {/* Requirements must always render so the Rule gets their performance values, so we need to toggle the items via display: none. I know it's not best practice, but it has very low cost and prevents a lot of trouble for contributors. */}
-          <div className="details" style={{ display: this.state.expanded ? undefined : 'none' }}>
+          <div className="details" style={{ display: this.state.expanded ? undefined : 'none', paddingLeft: 65, position: 'relative', minHeight: 80 }}>
```
[the remark this answers is not in the document]
Comment (rp4rk) [reply src/Parser/Core/Modules/Features/Checklist2/Rule.js]: Optimizing for readability, the icon is definitely not something I'd consider readable content. I'd consider the information icon to be there as anchorage for scanning, letting people know that this is content that is providing useful information.

**Review by MartijnHols — APPROVED** [review]

**Pull request merged**